src/50_Samples_%26_Templates/Product.html

```diff
@@ -313,11 +415,20 @@ <h5 class="price-description">Price: $1.99</h5>
         </div>
         <div>
           <label for="size">Size:</label>
-          <amp-selector name="size" layout="container">
+          <amp-selector name="size" layout="container" on="select:AMP.setState(selectedSize=event.targetOption)" [selected]="allColors[selectedCarousel].availableSizes.indexOf(selectedSize) >= 0 ? selectedSize : allColors[selectedCarousel].availableSizes[0]">
             <ul>
-              <li option="S">S</li>
-              <li option="M" selected>M</li>
+              <div class="size-container">
```

Comment:
Do we need the div here? I'm not sure if it's valid html if an `ul` is not followed by a `li`, but in any case we should keep the number of divs to a minimum to maintain readability.